src/test/unit/common/languageServer/languageServer.test.ts

```diff
@@ -49,6 +49,9 @@ suite('Language Server', () => {
           reportFalsePositives: false,
         };
       },
+      getFeaturesConfiguration() {
+        return { ossEnabled: true, codeSecurityEnabled: true, codeQualityEnabled: true, iacEnabled: true };
```

Comment:
Is it worth restructuring the tests to give a bit more context? At the moment, the test might seem confusing as all these values are true, but the test values are all false.

Most of these are hardcoded to false and `iacEnabled` is dependent on the feature flag too. Maybe consider structuring in the [BDD](https://mochajs.org/#bdd) style like [here](https://github.com/snyk/vscode-extension/blob/883dcb010333b2ae0ec1920f5062c3cccc7b6aeb/src/test/unit/common/proxy.test.ts#L36)?

Reply:
Actually these tests aren't needed any more. 

The code linked uses TDD interface, but you've probably meant test grouping, is it right?

Reply:
Right, the grouping structure rather than the interface used, so that it describes the intended behaviour for certain features, e.g. when ls feature toggle is false and iacEnabled is true; but if the tests aren't needed then nvm :)